src/SourceBuild/content/build.sh

```diff
@@ -10,34 +10,34 @@ set -e
 usage()
 {
   echo "Common settings:"
-  echo "  --binaryLog                  Create MSBuild binary log (short: -bl)"
-  echo "  --configuration <value>      Build configuration: 'Debug' or 'Release' (short: -c)"
-  echo "  --verbosity <value>          Msbuild verbosity: q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic] (short: -v)"
+  echo "  --binaryLog                     Create MSBuild binary log (short: -bl)"
+  echo "  --configuration <value>         Build configuration: 'Debug' or 'Release' (short: -c)"
+  echo "  --verbosity <value>             Msbuild verbosity: q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic] (short: -v)"
   echo ""
 
   echo "Actions:"
-  echo "  --clean                      Clean the solution"
-  echo "  --help                       Print help and exit (short: -h)"
-  echo "  --test                       Run smoke tests (short: -t)"
+  echo "  --clean                         Clean the solution"
+  echo "  --help                          Print help and exit (short: -h)"
+  echo "  --test                          Run smoke tests (short: -t)"
   echo ""
 
   echo "Source-only settings:"
-  echo "  --source-only                Source-build the solution (short: -so)"
-  echo "  --online                     Build using online sources"
-  echo "  --poison                     Build with poisoning checks"
-  echo "  --release-manifest <FILE>    A JSON file, an alternative source of Source Link metadata"
-  echo "  --source-repository <URL>    Source Link repository URL, required when building from tarball"
-  echo "  --source-version <SHA>       Source Link revision, required when building from tarball"
-  echo "  --use-mono-runtime           Output uses the mono runtime"
-  echo "  --with-packages <DIR>        Use the specified directory of previously-built packages"
-  echo "  --with-sdk <DIR>             Use the SDK in the specified directory for bootstrapping"
+  echo "  --source-only, --source-build   Source-build the solution (short: -so, -sb)"
```

Comment:
Shouldn't this be --sb?

Reply:
We already use `-sb` in all other repos that use Arcade and are part of source-build: https://github.com/dotnet/runtime/blob/52e1ad3779e57c35d2416cd10d8ad7d75b2c0c8b/eng/common/build.sh#L22

For POSX compliance we would need to change a few other options as well. The syntax is quite strict: https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html

I think we can live with the fact that our shell scripts in the VMR aren't fully POSIX compliant but it would for sure be nice to clean them up eventually.

Reply:
Aliases are always one `-`, right?